keybase/platform_windows.go

```diff
@@ -10,10 +10,22 @@ import (
 	"strings"
 	"time"
 
+	"github.com/kardianos/osext"
 	"github.com/keybase/go-updater"
 	"github.com/keybase/go-updater/command"
 )
 
+// execPath returns the app bundle path where this executable is located
+func (c config) execPath() string {
```

Comment:
i think this is destination path? can move to that?

you could nuke execPath too unless we need it... if so it should be the path to the executable that started the app